### PR TITLE
Cleanup Accumulo Admin utilities

### PR DESCRIPTION
**Pull request by cshannon**
Remove main methods from utilities now that they are exposed through
KeywordExecutable and the accumulo command. Also sort the commands in
the help section alphabetically.

### CODE REVIEW

**Comment by cshannon** [comment]
@milleruntime - This is ready for review and removes main methods and sorts the commands in help alphabetically.

**Review by DomGarguilo — CHANGES_REQUESTED** [review]
I think you may need to revise `KeywordStartIT.checkHasMain`. I ran it on this branch and it fails since some of the main methods were removed and that test expects them to be there.

**Comment by cshannon** [comment]
> I think you may need to revise `KeywordStartIT.checkHasMain`. I ran it on this branch and it fails since some of the main methods were removed and that test expects them to be there.

Ah yeah, thanks I forgot about that. I'll update it when I get a chance and run through the Sunny test profile to verify.

**Comment by cshannon** [comment]
KeywordStartIT has been fixed. The issue was I inadvertently removed the main method from the DumpZookeeper command which is still marked with KeywordExecutable and not part of the Admin command.

**Review by DomGarguilo — APPROVED** [review]
As far as I can tell, these changes look good :+1:

File: server/base/src/main/java/org/apache/accumulo/server/util/Admin.java [hunk 1]
```diff
@@ -226,49 +226,52 @@ public void execute(final String[] args) {
...
     DeleteZooInstanceCommand deleteZooInstanceOpts = new DeleteZooInstanceCommand();
     cl.addCommand("deleteZooInstance", deleteZooInstanceOpts);
 
-    RestoreZooCommand restoreZooOpts = new RestoreZooCommand();
-    cl.addCommand("restoreZoo", restoreZooOpts);
+    DumpConfigCommand dumpConfigCommand = new DumpConfigCommand();
+    cl.addCommand("dumpConfig", dumpConfigCommand);
 
     ListInstancesCommand listIntancesOpts = new ListInstancesCommand();
     cl.addCommand("listInstances", listIntancesOpts);
```
[the remark this answers is not in the document]
Comment (cshannon) [reply server/base/src/main/java/org/apache/accumulo/server/util/Admin.java]: @DomGarguilo - good catch, typo fixed. Is this good to merge or do you want to wait for @milleruntime to take a look too?

**Review by dlmarion — APPROVED** [review]

**Review by milleruntime — APPROVED** [review]

**Pull request merged**